minio/api.py

```diff
@@ -100,12 +94,21 @@ class Minio:  # pylint: disable=too-many-public-methods
     :return: :class:`Minio <Minio>` object
 
     Example::
-        client = Minio('play.min.io')
-        client = Minio('s3.amazonaws.com', 'ACCESS_KEY', 'SECRET_KEY')
-        client = Minio('play.min.io', 'ACCESS_KEY', 'SECRET_KEY',
-                       region='us-east-1')
+        # Create client with anonymous access.
+        client = Minio("play.min.io")
+
+        # Create client with access and secret key.
+        client = Minio("s3.amazonaws.com", "ACCESS-KEY", "SECRET-KEY")
+
+        # Create client with access key and secret key with specific region.
+        client = Minio(
+            "play.minio.io:9000",
+            access_key="Q3AM3UQ867SPQQA43P2F",
+            secret_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG",
+            region="my-region",
+        )
 
-    **NOTE on concurrent usage:** The `Minio` object is thread safe when using
+    **NOTE on concurrent usage:** `Minio` object is thread safe when using
```

Comment:
If my wording from [API.md](https://github.com/minio/minio-py/pull/1038/files#diff-b57590968a12cee85a37c1b91d8cc7092cd8b68e50b242c39a178121db82a797R25) looks OK, use the same wording here.